docker-compose.yml

```diff
@@ -1,11 +1,28 @@
 version: "3.8"
 
 services:
-  app:
-    #image: 
-    build: .
-    container_name: app-aquila
-    ports:
-      - "3005:3005"
+  #app:
+  #image:
+  #build: .
+  #container_name: app-aquila
+  #ports:
+  # - "3005:3005"
```

Comment:
pq isso ta comentado? se não for pra subir a alteração coloca isso em um no-commit list (chagelist nas ides da jetbrains) ou stash acho que rola também.